crates/sui-core/src/unit_tests/gas_tests.rs

```diff
@@ -384,7 +384,7 @@ async fn test_move_call_gas() -> SuiResult {
     let gas_used_before_vm_exec = gas_status.summary(true).gas_used();
     // The gas cost to execute the function in Move VM.
     // Hard code it here since it's difficult to mock that in test.
-    const MOVE_VM_EXEC_COST: u64 = 17006;
+    const MOVE_VM_EXEC_COST: u64 = 16006;
```

Comment:
This value seems to be arbitrary and have to change it for it to pass (it also has been changed in the past). Do we still need this test?

Reply:
That's strange. I don't expect this to change this much

Reply:
there is some wiggle room in comparison, +- 5000 should probably work as well.